pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -106,39 +116,28 @@ public static void updateOrderStatus(BaseMercuryStatusMessage baseMercuryStatusM
                             null, statusDate, null,
                             rs.getLong(DBConstants.TISSUE_ID), rs.getLong(DBConstants.DSM_KIT_REQUEST_ID), null));
                 } else {
-                    dbVals.resultException = new RuntimeException(
-                            "Couldn't get ddp instance id for order " + mercuryStatusMessage.getOrderID());
+                    dbVals.resultException = new DsmInternalError(String.format("Couldn't get ddp instance id for order %s",
+                            mercuryStatusMessage.getOrderID()));
                 }
             } catch (SQLException ex) {
                 dbVals.resultException =
-                        new RuntimeException("Error getting ddp instance id for order " + mercuryStatusMessage.getOrderID(), ex);
+                        new DsmInternalError(String.format("Error getting ddp instance id for order %s", mercuryStatusMessage.getOrderID()), ex);
```

Comment:
Is this error text correct? How does the DDP instance figure in here?

Reply:
I think someone did a wrong copy paste here long ago...